lib/mocks/billdispenser.js

```diff
@@ -1,9 +1,17 @@
-const _ = require('lodash/fp')
+const _ = require('lodash')
```

Comment:
We need to keep consistency with `lodash/fp`, otherwise the codebase will become very confusing. Always use `lodash/fp` and not `lodash`.